config_app/js/config-field-templates/config-numeric-field.html

```diff
@@ -1,6 +1,7 @@
 <div class="config-numeric-field-element">
   <form name="fieldform" novalidate>
     <input type="number" class="form-control" placeholder="{{ placeholder || '' }}"
-           ng-model="bindinginternal" ng-trim="false" ng-minlength="1" required>
+           ng-model="bindinginternal" ng-trim="false" ng-minlength="1" required
+           ng-readonly="isReadonly">
```

Comment:
Use [one-time binding](https://docs.angularjs.org/guide/expression#one-time-binding) to improve performance (`::isFieldReadonly(...)`).

Reply:
Thanks for the tip! Fixed